contracts/root/depositManager/DepositManagerStorage.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity ^0.5.5;
+
+import { Registry } from '../Registry.sol';
+import { RootChain } from '../RootChain.sol';
+import { ProxyData } from '../../common/misc/ProxyData.sol';
+
+contract DepositManagerStorage is ProxyData {
+  Registry internal registry;
```

Comment:
where is token mapping storage, maybe we can remove token manager now and just merge it with depositManager? @jdkanani

Reply:
I moved it to registry. Check it out if that looks fine.